### ISSUE TITLE
CMake Error

### ISSUE THREAD

**Issue by isuruf** [bluescarni/piranha#59]
```
CMake Error at cmake_modules/yacma/YACMAThreadingSetup.cmake:35 (try_compile):
  Attempt at a recursive or nested TRY_COMPILE in directory
    /home/travis/our_usr/piranha/build
Call Stack (most recent call first):
  CMakeLists.txt:50 (include)
```

Build log is here, https://travis-ci.org/isuruf/symengine/jobs/126044272#L381

**Comment by bluescarni** [comment]
Thanks for reporting the problem!

A tentative fix is available in #60.

**Comment by bluescarni** [comment]
@isuruf I have merged #60, let me know if it fixes the issue.

**Comment by isuruf** [comment]
It does. Thanks
